mkosi/__init__.py

```diff
@@ -3458,8 +3458,8 @@ def make_tar(args: CommandLineArguments, root: Path, do_run_build_script: bool,
 
 def find_files(root: Path) -> Generator[str, None, None]:
     """Generate a list of all filepaths relative to @root"""
-    root = str(root).rstrip("/") + "/"  # make sure the path ends in exactly one '/'
-    prefix_len = len(root)
+    root: str = str(root).rstrip("/") + "/"  # make sure the path ends in exactly one '/'
```

Comment:
We should just use a different variable name here

Reply:
Rewrote the function to return a `Path`, which then implies getting calling `os.fspath` on the results. In my tests the resulting lists are the same (except for the type), so this shouldn't break @keszybz's cpio archives.

Reply:
Still using the same variable name here though, can we fix that as well? Changing an existing variable's type just feels messy to me

Reply:
Are we looking at the same version? In the version here `root` isn't touched anymore, no?